src/Nox.Abstractions/Nox.Abstractions.csproj

```diff
@@ -42,5 +42,9 @@
     <ItemGroup>
       <ProjectReference Include="..\Nox.Types\Nox.Types.csproj" />
     </ItemGroup>
+
+    <ItemGroup>
+      <Folder Include="Application\Integrations\" />
```

Comment:
this should be removed, non existing folder

Reply:
Removed